src/translations/de.json

```diff
@@ -0,0 +1,30 @@
+{
+    "common": {
+        "name": "Meteoalarm-Karte",
+        "description": "Die Meteoalarm-Karte warnt dich vor bevorstehenden Wetterereignissen.",
+        "not_available": "Meteoalarm ist nicht verfügbar"
+    },
+    "error": {
+        "missing_entity": "Die angegebene Entität ist nicht vorhanden!"
+    },
+    "events": {
+        "no_warnings": "Keine Warnungen",
+        "wind": "Wind",
+        "snow_ice": "Schnee/Eis",
+        "thunderstorms": "Gewitter",
+        "fog": "Nebel",
+        "hight_temperature": "Extrem hohe Temperaturen",
+        "low_temperature": "Extrem niedrige Temperaturen",
+        "coastal_event": "Küstenereignis",
+        "forest_fire": "Waldbrand",
+        "avalanches": "Lawinen",
+        "rain": "Regen",
+        "flood": "Überflutung",
+        "rain_flood": "Regenüberflutung"
+    },
+    "messages": {
+        "yellow": "Warnstufe Gelb: {0}",
+        "orange": "Warnstufe Orange: {0}",
+        "red": "Warnstufe Rot: {0}"
```

Comment:
`messages` should feel like real sentences so remove `:`

Reply:
With that events above and considering the German language syntax, this would read very weird. No one would write or speak like this. So it's more natural to have the warning level prepended, followed by the event type that is going to happen.

Reply:
To be more clear: There are actually multiple ways of integrating a classification or rating combined with an event or explaination into a German sentence. I choose the one which is fastest to read for German users. Some more examples of how we could write this:

1. Gelbe Warnstufe: Gewitter
2. Gewitter (Warnstufe Gelb)
3. Gewitter der Warnstufe Gelb

All of above more or less translates into what you have in your English translation:
Yellow thunderstorm warning (which btw sound quite weird in English too, and I'd suggest asking a native speaker about this syntax).

After thinking about this, I will change this as ell to option 3.